plugins/outputs/splunk/README.md

```diff
@@ -0,0 +1,76 @@
+# Splunk Output Plugin
+
+This plugin writes metrics to a [Splunk Metric Index](https://docs.splunk.com/Documentation/Splunk/latest/Indexer/Setupmultipleindexes#Create_metrics_indexes) via a [Splunk HTTP Event Collector (HEC)](http://docs.splunk.com/Documentation/Splunk/latest/Data/UsetheHTTPEventCollector).  This requires an `Authorization Token` which will be created when you setup your Splunk HEC Token.
+
+Notes:
+1. If any point value cannot be converted to a float64, that metric will be skipped.
+2. Metrics are grouped by converting any `_` characters to `.` in the Metric Name.
+
+
+
+### Configuration:
+
+```toml
+# Configuration for Splunk server to send metrics to
+[[outputs.splunk]]
+  ## REQUIRED
+  ## URL of the Splunk Enterprise HEC endpoint (i.e.: http://localhost:8088/services/collector)
+  SplunkUrl = "http://localhost:8088/services/collector"
+
+  ## REQUIRED
+  ## Splunk Authorization Token for sending data to a Splunk HTTPEventCollector (HEC).
+  ##   Note:  This Token should map to a 'metrics' index in Splunk.  
+  AuthString = "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX"
+
+  ## OPTIONAL:  prefix for metrics keys
+  #prefix = "my.specific.prefix."
+
+  ## OPTIONAL:  whether to use "value" for name of simple fields
```

Comment:
What is a "simple field"?

Looking at the code, it seems that what this really does is that if not enabled, and the field name is "value", that the metric name in splunk is just the name of the measurement.
I think this could be made more clear as it took me a while to figure out what the feature was really doing.

Reply:
Added comments in the code as well as the readme.